llarp/router/router.hpp

```diff
@@ -242,15 +255,16 @@ namespace llarp
     NetworkConfig networkConfig;
     DnsConfig dnsConfig;
 
-    const IpAddress DefaultRPCBindAddr = IpAddress("127.0.0.1:1190");
+    const std::string DefaultRPCBindAddr = "tcp://127.0.0.1:1190";
     bool enableRPCServer = false;
-    std::unique_ptr<rpc::Server> rpcServer;
-    IpAddress rpcBindAddr = DefaultRPCBindAddr;
+    std::string rpcBindAddr = DefaultRPCBindAddr;
+    std::unique_ptr<rpc::RpcServer> m_RPCServer;
+
     const llarp_time_t _randomStartDelay;
 
-    /// lokid caller
-    std::unique_ptr<rpc::Caller> rpcCaller;
-    IpAddress lokidRPCAddr = IpAddress("127.0.0.1:22023");
+    std::shared_ptr<rpc::LokidRpcClient> m_lokidRpcClient;
+
+    std::string lokidRPCAddr = "ipc://loki.sock";
     std::string lokidRPCUser;
     std::string lokidRPCPassword;
```

Comment:
These later two aren't needed anymore.  (If you needed to do authentication you still can via a pubkey embedded in the lokidRPCAddr).